App.tsx

```diff
@@ -13,14 +13,8 @@ const App = () => {
 
   return (
     <View style={styles.container}>
-      <View style={styles.inputContainer}>
-        <TextInput style={styles.input} />
-        <CallingCodePicker
-          selectedValue={selectedCode}
-          onValueChange={setSelectedCode}
-          togglerContainerStyle={styles.togglerStyle}
-        />
-      </View>
+      <Text>selected code is: {selectedCode}</Text>
+      <CallingCodePicker onValueChange={setSelectedCode} />
```

Comment:
I think we don't need  the selected code prop. I keep the selected code and handle the filtering process in the component and return the calling code to the user.